test/unit/test-adapter.js

```diff
@@ -2,91 +2,63 @@
 
 var http = require('http');
 var assert = require('chai').assert;
-var proxyquire = require('proxyquire');
 var sinon = require('sinon');
 var nop = require('nop');
+var crypto = require('crypto');
+var Readable = require('stream').Readable;
 var getRandomPort = require('get-random-port');
+var errorCodes = require('../../dist/http-handler').errorCodes;
```

Comment:
i think this import goes away if you follow my advice about error codes above. it becomes a named import next to `systemUnderTest`.